experimental/devices/rainbowhat/example_test.go

```diff
@@ -0,0 +1,97 @@
+package rainbowhat_test
+
+import (
+	"fmt"
+	"image"
+	"image/color"
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"periph.io/x/periph/conn/gpio"
+	"periph.io/x/periph/conn/physic"
+	"periph.io/x/periph/experimental/devices/rainbowhat"
+
+	"periph.io/x/periph/host"
+)
+
+func main() {
```

Comment:
Did you mean to put this into cmd/ ? Because right now this won't show up as an example.